src/net/sourceforge/kolmafia/swingui/CoinmastersFrame.java

```diff
@@ -1432,6 +1432,12 @@ public DinostaurPanel() {
     }
   }
 
+  private class ReplicaMrStorePanel extends CoinmasterPanel {
```

Comment:
Good. I don't see where you instantiate the panel and add it t to the frame, though.